MSTR-110 : UI 수정 및 문제 풀이 페이지 다크모드 추가

Kim-Hyunjo opened the pull request:
### 작업 내역

> 구현 내용 및 작업 했던 내역

- [x] 문제 풀이 페이지, 로그인 모달 등 UI 수정
- [x] 문제 풀이 페이지 다크모드 추가
- [x] 패딩 작성 방법 수정
- [x] input 컴포넌트 type prop 받음 (html input tag attribute인 type)
- [x] searchbar  emotion -> vanilla-extract refactoring

### 작업 유형

- [x] 신규 기능 추가
- [ ] 버그 수정
- [x] 리펙토링
- [ ] 문서 업데이트

Kim-Hyunjo committed:
💄 Update Problem Detail Page UI

- padding 추가

Kim-Hyunjo committed:
＋ Add toggle button assets

Kim-Hyunjo committed:
+ Add asset moon

- for dark mode icon

Kim-Hyunjo committed:
✨ Add Darkmode in Question Detail Page

Kim-Hyunjo committed:
💄Add padding in login inputbox

Kim-Hyunjo committed:
🔥Remove unused assets

kshired commented on src/Page/QuestionDetailPage/style.css.ts:
```diff
@@ -21,7 +35,12 @@ export const pageStyle = style({
+  backgroundColor: vars.backgroundColor,
+
+  transition: 'background-color 0.2s',
+  animation: `${spread}`,
```
`  animation: spread` 불가능한가요? 위에 `backgroundColor: vars.backgroundColor` 보면 가능할 것 같아서요

Kim-Hyunjo replied:
수정해놓았습니다!!

kshired commented on src/Page/QuestionDetailPage/style.css.ts:
```diff
@@ -64,26 +83,31 @@ export const statisticsStyle = style({
+  padding: ' 0 1.5rem 0 1.5rem',
+  borderRadius: '10px',
+
+  // selectors: {
+  //   '& div': {
+  //     background: 'red',
+  //   },
+  // },
```
```suggestion
```

Kim-Hyunjo replied:
이부분도 삭제했습니다!

kshired commented on src/Page/QuestionDetailPage/style.css.ts:
```diff
@@ -102,8 +124,42 @@ export const problemDescContentStyle = style({
+});
+
+export const answerInputContentStyle = style({
+  'height': '90%',
+  'fontFamily': 'Inter',
+  'fontStyle': 'normal',
+  'fontWeight': '400',
+  'fontSize': '20px',
+  'lineHeight': '24px',
+
+  'color': vars.textColor,
+  'backgroundColor': vars.backgroundColor,
+
+  'width': '100%',
```
```suggestion
  height: '90%',
  fontFamily: 'Inter',
  fontStyle: 'normal',
  fontWeight: '400',
  fontSize: '20px',
  lineHeight: '24px',

  color: vars.textColor,
  backgroundColor: vars.backgroundColor,

  width: '100%',
```

kshired commented on src/Component/Box/InputBox/DefaultInputBox/style.css.ts:
```diff
@@ -21,8 +24,8 @@ export const defaultInputBoxStyle = style({
-  'display': 'flex',
-  'alignItems': 'center',
+
+  'padding': ' 0 1rem 0 1rem',
```
다른 코멘트에도 남겨놨지만, `'key' : 'value'` 와 `key : 'value'` 구조가 혼용되어 있어요.

Kim-Hyunjo replied:
이거 저도 이유를 궁금해하고있는 부분인데 vscode prettier auto fix로 저렇게 돼요 뭐라고 검색해야 되는지 모르겠어요... 최상단 key는 quote가 안붙고 하나 안쪽으로 들어가면 붙어요
![ezgif com-gif-maker (6)](https://user-images.githubusercontent.com/63906230/179964773-72ad9703-8246-4f62-8511-4d06e1d0101c.gif)

kshired replied:
@Kim-Hyunjo

맞는지는 모르겠지만..
<img width="676" alt="스크린샷 2022-07-20 오후 8 00 46" src="https://user-images.githubusercontent.com/36851531/179966540-ee9e1449-f207-4fc6-8ad6-d8b54f642da2.png">
 https://stackoverflow.com/questions/71963080/vs-code-how-to-remove-quotes-from-keys-only-but-value-remains

Kim-Hyunjo replied:
와!!!!!!!!!!!! 맞아요!!!!!!!!!!!!!!!!!!!!! 감사합니다!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!1

Kim-Hyunjo committed:
🔥 Remove unnecessary chars

Kim-Hyunjo committed:
🎨 Update prettier setting

- quote-props consistent to as-needed
- 한 오브젝트내에서 하나라도 quote를 필요로 하면 넣는 속성에서 꼭 필요한 곳에만 넣는 것으로 바꿈

kshired reviewed (approved):
LGTM!

Kim-Hyunjo committed:
💄 Change oauth logo file format

- from png to svg

Kim-Hyunjo committed:
Merge branch 'dev' into feat/110

kshired commented on src/Page/ResultPage/style.css.ts:
```diff
@@ -15,11 +15,21 @@ export const pageStyle = style({
 });
 
 export const pageContentStyle = style({
+<<<<<<< HEAD
   boxSizing: 'border-box',
   display: 'flex',
   alignItems: 'start',
   justifyContent: 'space-between',
   width: '100%',
+=======
+  'boxSizing': 'border-box',
+
+  'display': 'flex',
+  'alignItems': 'start',
+  'justifyContent': 'space-between',
+
+  'width': '100%',
+>>>>>>> dev
```
@Kim-Hyunjo 얘 conflict 발생했습니다. 수정해주세요

Kim-Hyunjo replied:
수정했슴미닼!

Kim-Hyunjo committed:
💥 Fix merge conflict

Kim-Hyunjo committed:
💄 Fix UI

- input box padding
- 회원 가입 페이지 높이 수정

kshired commented on src/Page/ResultPage/style.css.ts:
```diff
@@ -77,7 +68,7 @@ export const contentStyle = style({
   width: '100%',
   height: '60vh',
-  padding: '2rem',
+  padding: '1rem 2rem 1rem 2rem',
```
```suggestion
  padding: '1rem 2rem',
```

Kim-Hyunjo replied:
수정했습니당

Kim-Hyunjo committed:
🎨 Update padding code format

- top-left-bottom-right to vertical-horizontal

kshired reviewed (approved):
고생했습니다! ㅜㅜ 머지하고 쉬세여

Kim-Hyunjo committed:
✨ Add Password Input Box

Kim-Hyunjo committed:
♻️ Define input type

- remove password input box
- define input type

Kim-Hyunjo committed:
♻️ Refactor Searchbar

- from emotion to vanilla-extract

Kim-Hyunjo committed:
🏷 Fix typos

Kim-Hyunjo commented:
> 고생했습니다! ㅜㅜ 머지하고 쉬세여

시러요‼️

kshired commented on src/Page/JoinPage/index.tsx:
```diff
@@ -14,11 +14,19 @@ function JoinPage() {
+          <DefaultInputBox
+            id='password-confirm'
+            placeholder='비밀번호를 다시 한번 입력해주세요'
+            type='password'
```
```suggestion
            type={INPUT_TYPE.PASSWORD}
```

Kim-Hyunjo committed:
Update src/Page/JoinPage/index.tsx

Co-authored-by: SEONGILKIM <shiroed1211@naver.com>

kshired reviewed (approved):
LGTM
